src/components/summaryDetail/Instructions.jsx

```diff
@@ -0,0 +1,17 @@
+import { nanoid } from 'nanoid';
+
+export default function Instructions({ instructions }) {
+  const instructionEls = instructions.map((instruction, index) => (
+    <p key={nanoid()} className="pt-4 text-xs">
```

Comment:
As Robester mentioned in the code review of my code, "I assume nanoid is a random key generator. It is not recommended to use a random key generator as a key. You should use some sort of combination of strings from the recipes object or a unique id provided by the backend, which should also be in the recipes object."

Reply:
Addressed with commit [919f74e](https://github.com/chingu-voyages/v46-tier2-team-20/pull/51/commits/919f74e0503b9e9cbe3044503f0b9f3e9f13b4f4)